src/indigo/less/type.less

```diff
@@ -7,10 +7,6 @@ body,
     font-weight: 300;
 }
 
-strong {
-    font-weight: 400;
```

Comment:
Nie je lepsie tam dat 600? takto to bude mat default (700) a taky rez tam nemame. Nevadi to?

Reply:
Jo, mas pravdu. Radsi jsem to jeste checknul. Myslel jsem ze si to vydefinujes ve font-face. U techto konkretnich fontu to tak neni. fixnu